### ISSUE TITLE
Remove underscores from Custom Metadata Type API names.

### ISSUE THREAD

**Issue by sakarettm07** [mattsimonis/boostr#31]
When entering the Label for a new Custom Metadata Type, the API Name is auto-populated with underscores.

<img width="914" alt="screen shot 2017-01-12 at 10 34 57 am" src="https://cloud.githubusercontent.com/assets/6854330/21896022/d073e88c-d8b2-11e6-925e-cdc6dad30edd.png">

**Comment by rocnick** [comment]
I would request that this be an optional setting.

**Comment by derekhansen** [comment]
@rocnick agreed, currently for objects and fields it's controlled by the Object/Field API Name option for boostr.
